ansible/library/fabric_info.py

```diff
@@ -48,8 +48,8 @@ def main():
         num_fabric_asic = int( m_args[ 'num_fabric_asic' ] )
         v4pfx = str( m_args[ 'asics_host_basepfx' ] ).split("/")
         v6pfx = str( m_args[ 'asics_host_basepfx6' ] ).split("/")
-        v4base = int( ipaddress.IPv4Address(v4pfx[0]) )
-        v6base = int( ipaddress.IPv6Address(v6pfx[0]) )
+        v4base = int( ipaddress.IPv4Address(unicode(v4pfx[0])) )
+        v6base = int( ipaddress.IPv6Address(unicode(v6pfx[0])) )
```

Comment:
FYI - this fix is already in https://github.com/Azure/sonic-mgmt/pull/3746

Reply:
@sanmalho-git, #3746 is related to multi-asic linecard and this change is related to supervisor linecard
Can we push this change in this PR.

Reply:
sure - i am ok.